docs/reference/query-dsl/bool-query.asciidoc

```diff
@@ -81,7 +81,7 @@ all documents where the `status` field contains the term `active`.
 This first query assigns a score of `0` to all documents, as no scoring
 query has been specified:
 
-[source,json]
+[source,js]
```

Comment:
Do you think we ought to complain for anything `[source,json]`? It'd be super easy to print a warning/fail the build.